jobs/jobs/airflow_utils.py

```diff
@@ -0,0 +1,94 @@
+import dataclasses
+import os
+from typing import Any, Dict, Iterable, List
+
+import airflow_client.client as client
+from airflow_client.client.api.dag_api import DAGApi
+from airflow_client.client.api.dag_run_api import DAGRunApi
+from airflow_client.client.model.dag_run import DAGRun
+
+import jobs.pipeline as pipeline
+
+AIRFLOW_USERNAME = os.getenv("AIRFLOW_USERNAME")
+AIRFLOW_PASSWORD = os.getenv("AIRFLOW_PASSWORD")
+AIRFLOW_SERVICE_PORT = os.getenv("AIRFLOW_SERVICE_PORT")
+
+AIRFLOW_URL = (
+    f"{os.getenv('AIRFLOW_SERVICE_SCHEME')}://"
+    f"{os.getenv('AIRFLOW_SERVICE_HOST')}"
+)
+if AIRFLOW_SERVICE_PORT:
+    AIRFLOW_URL = f"{AIRFLOW_URL}:{AIRFLOW_SERVICE_PORT}"
+
+AIRFLOW_URL = f"{AIRFLOW_URL}{os.getenv('AIRFLOW_SERVICE_PATH_PREFIX')}"
+
+
+def get_configuration():
+    return client.Configuration(
+        username=AIRFLOW_USERNAME,
+        password=AIRFLOW_PASSWORD,
+        host=AIRFLOW_URL,
+    )
+
+
+def get_api_instance():
+    configuration = get_configuration()
+
+    with client.ApiClient(configuration) as api_client:
+        yield api_client
```

Comment:
I didn't notice any usage of this function anywhere in the code. Do we need it?

Reply:
Let's keep it until we finish airflow integration